strings-csharp/ReplaceSpecialCharactersInString/ReplaceSpecialCharactersInString/StringReplacerBenchmarks.cs

```diff
@@ -0,0 +1,105 @@
+﻿using BenchmarkDotNet.Attributes;
+
+namespace ReplaceSpecialCharactersInString
+{
+    [MemoryDiagnoser]
+    public class StringReplacerBenchmarks
+    {
+        private string originalString = "";
+
+        private char OldChar { get; set; } = '#';
```

Comment:
follow proper naming convention for class property name and consistency. We can just declare all private properties first and assign value in Setup method.